cni/network/invoker_azure.go

```diff
@@ -95,14 +96,17 @@ func (invoker *AzureIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, er
 			nwCfg6.IPAM.Subnet = invoker.nwInfo.Subnets[1].Prefix.String()
 		}
 
-		addResult.ipv6Result, err = invoker.plugin.DelegateAdd(nwCfg6.IPAM.Type, &nwCfg6)
+		var ipv6Result *cniTypesCurr.Result
+		ipv6Result, err = invoker.plugin.DelegateAdd(nwCfg6.IPAM.Type, &nwCfg6)
 		if err != nil {
 			err = invoker.plugin.Errorf("Failed to allocate v6 pool: %v", err)
+		} else {
+			addResult.defaultCniResult.ipResult.IPs = append(addResult.defaultCniResult.ipResult.IPs, ipv6Result.IPs...)
```

Comment:
can create a temp variable of this type `addResult.defaultCniResult.ipResult`  and later assign it to addResult.defaultcniresult since referring this in multiple places

Reply:
just re-using ipv4Result and moved assignment to CNIResult at the end